java/rocksjni/portal.h

```diff
@@ -4474,25 +4458,25 @@ class JniUtil {
       return jbyte_strings;
     }
 
-    /**
-     * Copies bytes from a rocksdb::Slice to a jByteArray
-     *
-     * @param env A pointer to the java environment
-     * @param bytes The bytes to copy
-     *
-     * @return the Java byte[] or nullptr if an exception occurs
-     */
-    static jbyteArray copyBytes(JNIEnv* env, const Slice& bytes) {
-      const jsize jlen = static_cast<jsize>(bytes.size());
-
+    static jbyteArray createJavaByteArrayWithSizeCheck(JNIEnv* env, const char* bytes, const size_t size) {
```

Comment:
Could you add some function documentation please?
In particular can you add an `@throws`.

I have been using the Javadoc style even though it is C++, as there are some C++ documenters that will happily read javadoc style comments from C++.